packages/pulpcore-selinux/pulpcore-selinux.spec

```diff
@@ -16,7 +16,7 @@ BuildRequires:  checkpolicy
 BuildRequires:  selinux-policy-devel
 BuildRequires:  systemd
 Requires:       selinux-policy >= %{_selinux_policy_version}
-Requires:       pulpcore
+Requires:       pulpcore >= 3.73
```

Comment:
Should we require it via `python%{pulpcore_python}dist(pulpcore)` so in the future the Python version will always match? We also have the `fixfiles` invocations and I wonder if we could match those. Perhaps via a dedicated macro that we use both here and the `fixfiles` calls.

Reply:
`fixfiles` only knows about (real) rpm names (it does `rpm -q` under the hood), so it has to be `pythonX.Y-pulpcore`.
given we have the python version as a macro, also using `pythonX.Y-pulpcore` here would also work (and would have exposed the issue earlier!)